rune/libenclave/attestation/internal/sgx/pccs/pccs.go

```diff
@@ -0,0 +1,115 @@
+package pccs // import "github.com/inclavare-containers/rune/libenclave/attestation/internal/sgx/pccs"
```

Comment:
Actually you don't know anything about pccs because QVL encapsulates it under QPL. So please rename the package name to sgx/dcap/qvl.go